src/Core/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommand.cs

```diff
@@ -59,30 +49,27 @@ public async Task UpdateSecretsManagerSubscription(SecretsManagerSubscriptionUpd
 
         var plan = GetPlanForOrganization(organization);
 
-        Tuple<int, int>? adjustSeatsResponse = null;
-        Tuple<int, int>? adjustServiceAccountsResponse = null;
-
-        if (update.SeatAdjustment != 0)
+        if (update.AdjustingSeats)
         {
-            adjustSeatsResponse = await AdjustSeatsAsync(organization, update, plan);
+            await AdjustSeatsAsync(organization, update, plan);
         }
 
-        if (update.ServiceAccountsAdjustment != 0)
+        if (update.AdjustingServiceAccounts)
         {
-            adjustServiceAccountsResponse = await AdjustServiceAccountsAsync(organization, update, plan);
+            await AdjustServiceAccountsAsync(organization, update, plan);
         }
 
-        if (update.MaxAutoscaleSeats.HasValue && update.MaxAutoscaleSeats != organization.MaxAutoscaleSmSeats.GetValueOrDefault())
+        if (update.AutoscaleSeats)
         {
             UpdateSeatsAutoscaling(organization, update.MaxAutoscaleSeats.Value, plan);
```

Comment:
We could change this to get rid of the github warning
```suggestion
            UpdateSeatsAutoscaling(organization, update.MaxAutoscaleSeats.GetValueOrDefault(), plan);
```

Reply:
sure